### DISCUSSION TITLE
Add visual cue for multipart/form-data endpoints

### DISCUSSION THREAD

**Discussion by thelinuxlich** [scalar/scalar#551] [Ideas]
It would be nice if Scalar added something visual to identify when the endpoint can receive a file

**Comment by hanspagel** [comment]
True :) Some specs define multiple request/responses types. I’d like to add tabs to the responses, and probably to the requests too. Wdyt?

**Reply by thelinuxlich** [reply]
I don't know what you mean yet, responses already have tabs 
![image](https://github.com/scalar/scalar/assets/5043/8ffa05ac-091c-48a9-a224-ede12e3b141b)
